PluginBuilder/Controllers/AdminController.cs

```diff
@@ -351,18 +359,25 @@ public async Task<IActionResult> EmailSender(EmailSenderViewModel model)
         
         try
         {
-            await emailService.SendEmail(model.To, model.Subject, model.Message);
-            TempData[TempDataConstant.SuccessMessage] = $"Test email sent successfully to {model.To}.";
+            var emailsSent = await emailService.SendEmail(model.To, model.Subject, model.Message);
+            TempData[TempDataConstant.SuccessMessage] = $"Emails sent successfully to {emailsSent.Count} recipient(s).";
         }
         catch (Exception ex)
-        {
+        { 
             ModelState.AddModelError(string.Empty, $"Failed to send test email: {ex.Message}");
```

Comment:
We don't have Error Summary on view, so this error will not be displayed. Better use TempData Warning message